.github/workflows/ci.yml

```
@@ -99,5 +99,7 @@ jobs:
         if: always()
         with:
           name: rally-tracks-compat-logs
-          path: /home/runner/.rally/logs/
+          path: |
+            /home/runner/.rally/logs/
+            /home/runner/.rally/benchmarks/races/
```

What about            
```suggestion
            /home/runner/.rally/benchmarks/races/*/*/*/*/rally-benchmark.log
```

That way we don't include the race.json files

---

Addressed in https://github.com/elastic/rally-tracks/pull/442/commits/270eedc4b817273ec30a6b1ebec097ea0c1d407c.